baseapp/baseapp.go

```diff
@@ -71,6 +71,9 @@ type BaseApp struct {
 	// minimum fees for spam prevention
 	minimumFees sdk.Coins
 
+	// tx sigs limit for spam prevention
+	txSigLimit int
```

Comment:
This is in the consensus state machine, it has to be constant (not configurable)